docs/specs/app2app.md

```diff
@@ -21,20 +21,38 @@ An app can start the authentication flow by opening a link to another app, inste
 
 ### Server-side
 
-- Authorization Endpoint
+- Configuration
 
-  - A new parameter `device_key` is supported. This value can be specified by the client to enable app2app login. If specified, it should be a public key of a key pair, which the private key should be stored in a secure storage in the device. Such as the Android keystore or the iOS keychain.
-    - Multiple key types such as RS256 and NIST P-256 are supported.
+  - A app2app section will be added to the client configs.
+
+  ```yaml
+  oauth:
+    clients:
+      - client_id: CLIENT_ID
+        # ...Other configs
+        x_app2app:
+          enabled: true
+          allow_insecure_device_key_binding: false
```

Comment:
The convention in client metadata is not to use nested object. So I think we should call them `x_app2app_enabled` and `x_app2app_insecure_device_key_binding_enabled`.